register_and_run.sh

```diff
@@ -134,7 +140,10 @@ _getTerminationSignal() {
 trap _getTerminationSignal TERM
 
 # Register the runner
-gitlab-runner register
+gitlab-ci-multi-runner register \
+  --name ${MESOS_TASK_ID} \
+  --docker-privileged \
+  --docker-volume-driver ${DOCKER_VOLUME_DRIVER}
 
 # Start the runner
-gitlab-runner run --working-directory=${RUNNER_WORK_DIR} --metrics-server $HOST:$PORT0
+gitlab-ci-multi-runner run --user=gitlab-runner --working-directory=${RUNNER_WORK_DIR} --metrics-server :$PORT0
```

Comment:
@tobilg please don't merge it, because it is working wrong for a dind. Looks like gitlab or docker bug.  

P.S It was removed because runner can't start with specified host